src/SamplesApp/SamplesApp.netcoremobile/SamplesApp.netcoremobile.csproj

```diff
@@ -50,13 +52,6 @@
 
 		<When Condition="'$(TargetPlatformIdentifier)'=='ios'">
 			<PropertyGroup Condition="'$(TargetPlatformIdentifier)'=='ios'">
-				<MtouchExtraArgs>$(MtouchExtraArgs) --setenv=MONO_GC_PARAMS=soft-heap-limit=512m,nursery-size=64m,evacuation-threshold=66,major=marksweep,concurrent-sweep</MtouchExtraArgs>
-
-				<!-- See https://github.com/unoplatform/uno/issues/9430 for more details. -->
-				<MtouchExtraArgs>$(MtouchExtraArgs) --registrar:static</MtouchExtraArgs>
-
-				<!-- https://github.com/xamarin/xamarin-macios/issues/14812 -->
-				<MtouchExtraArgs>$(MtouchExtraArgs) --marshal-objectivec-exceptions:disable</MtouchExtraArgs>
```

Comment:
^ that does not seems related to the commit message

> Add support for fonts in Assets

The last two entries could be removed for `net7.0` but not for `net6.0`.